scout/parse/variant/genotype.py

```diff
@@ -324,14 +326,14 @@ def get_ref_depth(
     """Get reference read depth"""
     ref_depth = int(variant.gt_ref_depths[pos])
     if ref_depth == -1:
-        if any([paired_end_ref, split_read_ref]):
```

Comment:
The problem with this code is that any will be False even if one of these variables is present but is 0